packages/system/keycloak-configure/templates/configure-kk.yaml

```
@@ -84,3 +90,154 @@ spec:
+    name: keycloakrealm-cozy
+    kind: ClusterKeycloakRealm
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: tenant-root-dashboard-resources
+  namespace: tenant-root
+rules:
+- apiGroups:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)